web-api/migration-terraform/main/lambdas/migrations/0005-block-case-with-legacy-served-and-pending-items.js

```diff
@@ -0,0 +1,113 @@
+const createApplicationContext = require('../../../../src/applicationContext');
+const {
+  aggregateCaseItems,
+} = require('../../../../../shared/src/persistence/dynamo/helpers/aggregateCaseItems');
+const {
+  AUTOMATIC_BLOCKED_REASONS,
+} = require('../../../../../shared/src/business/entities/EntityConstants');
+const {
+  Case,
+} = require('../../../../../shared/src/business/entities/cases/Case');
+const {
+  createISODateString,
+} = require('../../../../../shared/src/business/utilities/DateHandler');
+const { cloneDeep } = require('lodash');
+
+const applicationContext = createApplicationContext({});
+
+const migrateItems = async (items, documentClient) => {
```

Comment:
The nested if/elses along with the size of this code block make it hard to read - it would be nice to break these kinds of blocks out into functions.